lib/hanami/validations.rb

```diff
@@ -204,7 +204,7 @@ def messages(type)
       #   class Signup
       #     include Hanami::Validations
       #
-      #     messages :i18n
+      #     messages_path 'config/messages.yml'
```

Comment:
If you provide a path then it'll be used to load custom messages that will be merged with the builtin ones thus it shouldn't be required. If it is, I'd say it's probably a bug.